firmware/Core/Src/gps/gps_internal_drivers.c

```diff
@@ -0,0 +1,144 @@
+#include "main.h"
+
+#include "debug_tools/debug_uart.h"
+#include "debug_tools/debug_i2c.h"
+#include "gps/gps_internal_drivers.h"
+#include "uart_handler/uart_handler.h"
+#include "stm_drivers/timing_helpers.h"
+#include "uart_handler/uart_handler.h"
+#include "log/log.h"
+
+#include <stdint.h>
+#include <string.h>
+
+extern UART_HandleTypeDef *UART_gps_port_handle;
+
+const uint32_t GPS_RX_TIMEOUT_MS = 800;
```

Comment:
Please split into two separate variables - timeout before first byte, and timeout between bytes.

Reply:
Done. For now they have the same time value and I'll update them with the appropriate values after ive checked through the documentation